features/api/student-convention.feature

```diff
@@ -1,38 +1,43 @@
-@job
-Feature: Jobs management
-  The job of a user is the job he has occupied for a given mandate.
-  A job must belong to a mandate.
-  At each new mandate, the jobs available are the job enabled.
-  A job may be left empty.
-  New jobs are created for the current mandate.
+
+Feature: Student-convention
+  this refere to an offial document 
```

Comment:
we should give some information as its relation with the student, like "a user may have a student convention"